packages/frontend/src/components/organisms/MemberForm.tsx

```diff
@@ -98,9 +134,23 @@ export const MemberForm = (props: IProps) => {
         setFieldError('global', 'Erreur serveur...');
       }
     }
-
     setSubmitting(false);
   };
+  const backToMemberPage = () => {
+    history.push('/admin/members');
+  };
+
+  const changePlanSelected = (id: string) => {
+    setPlanSelectedId(id);
+  };
+
+  const updateMode = () => {
+    if (props.member?.id) {
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
Can be simplified into ```()=>props.member?.id===true ```

Reply:
typescript won't like it, how to do it typescript friendly ?